scripts/publish.js

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env node
```

Comment:
At term, this script has clearly a better place in `cozy-app-publish` as a new mode IMHO